### PR TITLE
Font sizes updated

### PR DESCRIPTION
**Pull request by rkzel**
This addresses all small updated mentioned in the associated ticket.

### CODE REVIEW

**Review by chadoh — CHANGES_REQUESTED** [review]
I would rather we stick with aragonUI standard components as much as possible, even if it results in a slight divergence from our original design. Benefits:

* when aragonUI gets updated, our interface is automatically updated to match
* a more standard look/feel across aragon apps

@stellarmagnet what do you think? Most of the changes here cause our Profile UI to diverge from aragonUI standards.

File: src/client/components/EducationHistoryTile.js [hunk 1]
```diff
@@ -13,8 +13,16 @@ const EducationHistoryTile = ({
 }) => (
   <SingleEducationItem>
     <div>
-      <Text.Block size="large">{educationHistoryData.organization}</Text.Block>
-      <Text.Block size="normal">
+      <Text.Block
+        size="large"
+        css={`
+          line-height: 1.8;
+          font-weight: bold;
```
Comment (chadoh) [inline src/client/components/EducationHistoryTile.js]: Do we need this?

File: src/client/components/informationPanel/InformationCard.js [hunk 2]
```diff
@@ -91,6 +91,7 @@ const StyledCard = styled(Card).attrs({ width: '100%', height: 'auto' })`
 const EthAddr = styled(Text).attrs({ size: 'small' })`
   color: ${theme.textTertiary};
   word-break: break-all;
+  font-size: 12px;
```
Comment (chadoh) [inline src/client/components/informationPanel/InformationCard.js]: Can we change the `attrs` to send the appropriate `size` prop to `<Text>` and avoid this override?

File: src/client/wrappers/styleWrappers/CardWrapper.js [hunk 3]
```diff
@@ -1,36 +1,20 @@
...
-      <Text
-        css={`
-          padding: 7px 0;
-        `}
-        size="xlarge"
-      >
+      <Text css="padding: 7px 0" size="xlarge">
         {title}
       </Text>
     )}
```
Comment (chadoh) [inline src/client/wrappers/styleWrappers/CardWrapper.js:13]: This seems short enough now to put on one line:

    {title && <Text css="padding: 7px 0" size="xlarge">{title}</Text>}

File: src/client/components/WorkHistoryTile.js [hunk 4]
```diff
@@ -12,29 +12,19 @@ const WorkHistoryTile = ({ workHistoryData, openModal, removeItem }) => (
       <Text.Block
         size="large"
         css={`
-          font-weight: 700;
+          line-height: 1.8;
+          font-weight: bold;
```
Comment (chadoh) [inline src/client/components/WorkHistoryTile.js]: Same question here: do we need this?

And it looks like we're overriding the styles of `Text.Block` in the same way in two places. If we do need these overrides, is it time to extract a new component?

**Comment by rkzel** [comment]
# Before
56-before

# After
56-after

File: src/client/components/informationPanel/InformationCard.js (hunk 2 above)
Comment (rkzel) [reply src/client/components/informationPanel/InformationCard.js]: Yes!

File: src/client/wrappers/styleWrappers/CardWrapper.js (hunk 3 above)
Comment (rkzel) [reply src/client/wrappers/styleWrappers/CardWrapper.js:13]: Seems so, but it gets highlighted by eslint if it's in one line.

File: src/client/components/EducationHistoryTile.js (hunk 1 above)
Comment (rkzel) [reply src/client/components/EducationHistoryTile.js]: That made a subtle difference as requested, yes, both lines.

File: src/client/components/WorkHistoryTile.js (hunk 4 above)
Comment (rkzel) [reply src/client/components/WorkHistoryTile.js]: I draw my line at 3 occurrences ;) But good point, I added new component for them.

File: src/client/components/styled-components/index.js [hunk 5]
```diff
@@ -103,6 +103,20 @@ export const DropDownWithValidation = props => (
 DropDownWithValidation.propTypes = { error: PropTypes.string }
 DropDownWithValidation.defaultProps = { error: '' }
 
+export const SchoolCompanyName = props => (
```
Comment (chadoh) [inline src/client/components/styled-components/index.js]: Maybe `CardHeader` would be a better name?

**Review by chadoh — APPROVED** [review]
Nice cleanup! I'd like to see two last small changes before merge, but I don't need to re-review once they're implemented, so I'm going ahead and marking "approved" now.

1. Rename the new `SchoolCompanyName` component, as suggested in my comment below
2. Squash the commits together. I don't think having commits in our history with messages like "experiments" will help us.

File: src/client/components/styled-components/index.js (hunk 5 above)
Comment (rkzel) [reply src/client/components/styled-components/index.js]: Much better! Although it's TileHeader, Card contains multiple Tiles, and this is single Tile header.

Comment (rkzel) [reply src/client/components/styled-components/index.js]: Second point: yes, we usually do "squash and merge" in github while merging.

**Pull request merged**